app/src/main/java/com/orange/ouds/app/ui/MainScreen.kt

```diff
@@ -83,7 +83,7 @@ fun MainScreen(mainViewModel: MainViewModel = hiltViewModel()) {
     var changeThemeDialogVisible by remember { mutableStateOf(false) }
 
     OudsTheme(
-        themeContract = mainState.themeState.currentTheme,
+        theme = mainState.themeState.currentTheme,
```

Comment:
This is not visible to users of our library, but there is a reference to `themeContract` in `PreviewCodeSnippet` in `CodeSnippet.kt`.